week-4/A-objects-intro/exercise-part-0.js

```diff
@@ -4,4 +4,16 @@ Describe your own laptop as a JavaScript object
 
 Try to think of as many properties as you can!
 
-*/
\ No newline at end of file
+*/
+
+var laptop = {
+  Brand: "Lenovo",
+  Color: "black",
+  touchscreen: "false",
+  Memory: "3.6 GIB",
+  Processor: "Intel® Core™ i5-3320M CPU @ 2.60GHz × 4",
+  Grapichs: "ntel® Ivybridge Mobile",
+  GNOME: "3.30.1",

```

Comment:
Just a detail: properties in JavaScript all always camel case. So you'd have `gnome` and `osType`. Consistency is important.